statistics-reports.md

```diff
@@ -0,0 +1,202 @@
+# Displaying usage statistics
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Statistics for a DSpaceObject
+**GET /api/statistics/usage-reports**
+
+As we don't have a use case to iterate over all the existing usage reports, the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error-codes).
+
+## Single statistic
+**GET /api/statistics/usage-reports/<:id>**
+
+This endpoint provides a specific statistic
+
+An example JSON response document to `/api/statistics/usage-reports/1911e8a4-6939-490c-b58b-a5d70f8d91fb_TopCountries`:
+```json
+{
+    "id": "1911e8a4-6939-490c-b58b-a5d70f8d91fb_TopCountries",
+    "type": "usage-report",
+    "report-type": "TotalVisits",
+    "points": [
+        {
+            "label": "United States",
+            "type": "country",
+            "id": "US",
+            "values": {
+                "views": 2
+            }
+        },
+        {
+            "label": "China",
+            "type": "country",
+            "id": "CN",
+            "values": {
+                "views": 1
+            }
+        }
+    ]
+}
+```
+
+Possible response status
+
+- 200 OK - The specific statistics data was found, and the data has been properly returned.
+* 401 Unauthorized - if you are not authenticated and the statistics are not available to the Anonymous user
+* 403 Forbidden - if you are not logged in with sufficient permissions and the statistics are not available to the Anonymous user
+- 404 Not Found - The specified ID was not found
+
+## Search Statistics for a DSpaceObject
+**GET /api/statistics/usage-reports/search/object**
+
+This endpoint provides a paginated list of statistics for a DSpaceObject. 
+
+The DSpaceObject is given through the following parameters:
+- `scope` The UUID of the DSpaceObject
+- `scopeType` The string type of the DSpaceObject 
+
+The usual parameters for paginated lists are supported as well:
+- `page` The page number 
+- `size` The number of statistics in a page
```

Comment:
do you mean the number of report, right? this is not related in any way to the number of points returned, right?

Reply:
That's correct.
I've been wondering whether we should support changing the number of points, but only if we'd use a "usage-report" with paging because there can be multiple for an item, and we'd use e.g. a "usage-value" with paging to represent the rows in the table also with paging. I didn't include it because it's not supported in DSpace 6.